patches/third_party-blink-renderer-modules-payments-payment_item.idl.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/third_party/blink/renderer/modules/payments/payment_item.idl b/third_party/blink/renderer/modules/payments/payment_item.idl
+index 48b92d94e24b17c9219992a34dbfe178164f1639..8f5af36ce69ea5de14cbbf6f2bdf80ac3b001923 100644
+--- a/third_party/blink/renderer/modules/payments/payment_item.idl
++++ b/third_party/blink/renderer/modules/payments/payment_item.idl
+@@ -7,5 +7,6 @@
+ dictionary PaymentItem {
+     required DOMString label;
+     required PaymentCurrencyAmount amount;
++    DOMString sku;
```

Comment:
what is sku vs label? Do we need both or can we use label in place of sku or can we make this a json string and put both in there? I'd like to avoid the patches, but if there's a good reason to have a separate field then I'm ok with that

Reply:
`label` is used by the currently supported payment methods like `basic-card` and `service-worker`.  It is the display name of the item that is displayed on the `checkout modal`. I added a different field so developers can use the same `display` object for all payment methods.